### PR TITLE
Make email display in footer and contact us page configurable

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | dev
| Description?  | Make the display of email address in footer and contact page's left column configurable. 
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #14416
| How to test?  | see #14416

### CODE REVIEW

**Comment by matthieu-rolland** [comment]
hi @LouiseBonnard , this PR has a wording, I ping you here because it doesn't seem like there is a wording label on this repository...

File: ps_contactinfo.php [hunk 1]
```diff
@@ -116,4 +117,59 @@ public function hookActionAdminStoresControllerUpdate_optionsAfter()
...
+
+            Tools::redirectAdmin($this->context->link->getAdminLink('AdminModules', true).'&conf=6&configure='.$this->name.'&tab_module='.$this->tab.'&module_name='.$this->name);
+        }
+
+        $helper = new HelperForm();
+        $helper->submit_action = 'submitContactInfo';
+
+        $field = array(
+            'type' => 'switch',
+            'label' => $this->trans('Display email address', array(), 'Admin.Global'),
```
Comment (LouiseBonnard) [inline ps_contactinfo.php]:
```suggestion
            'label' => $this->trans('Display email address', array(), 'Admin.Actions'),
```

File: ps_contactinfo.php [hunk 2]
```diff
@@ -116,4 +117,59 @@ public function hookActionAdminStoresControllerUpdate_optionsAfter()
...
+            'name' => 'PS_CONTACT_INFO_DISPLAY_EMAIL',
+            'values' => array(
+                array(
+                    'id' => 'active_on',
+                    'value' => 1,
+                    'label' => $this->trans('yes', array(), 'Admin.Global')
```
Comment (LouiseBonnard) [inline ps_contactinfo.php]:
```suggestion
                    'label' => $this->trans('Yes', array(), 'Admin.Global')
```

File: ps_contactinfo.php [hunk 3]
```diff
@@ -116,4 +117,59 @@ public function hookActionAdminStoresControllerUpdate_optionsAfter()
...
+                ),
+                array(
+                    'id' => 'active_off',
+                    'value' => 0,
+                    'label' => $this->trans('no', array(), 'Admin.Global')
```
Comment (LouiseBonnard) [inline ps_contactinfo.php]:
```suggestion
                    'label' => $this->trans('No', array(), 'Admin.Global')
```

File: ps_contactinfo.php (hunk 1 above)
Comment (matthieu-rolland) [reply ps_contactinfo.php]: done

File: ps_contactinfo.php (hunk 2 above)
Comment (matthieu-rolland) [reply ps_contactinfo.php]: done

File: ps_contactinfo.php (hunk 3 above)
Comment (matthieu-rolland) [reply ps_contactinfo.php]: done

File: ps_contactinfo.php [hunk 4]
```diff
@@ -116,4 +117,59 @@ public function hookActionAdminStoresControllerUpdate_optionsAfter()
 
         return true;
     }
+
+    public function getContent()
+    {
+        $output = '';
```
Comment (PierreRambaud) [inline ps_contactinfo.php]:
```suggestion
        $output = [];
```

File: ps_contactinfo.php [hunk 5]
```diff
@@ -116,4 +117,59 @@ public function hookActionAdminStoresControllerUpdate_optionsAfter()
...
+                    'id' => 'active_off',
+                    'value' => 0,
+                    'label' => $this->trans('No', array(), 'Admin.Global')
+                )
+            )
+        );
+
+        $helper->fields_value['PS_CONTACT_INFO_DISPLAY_EMAIL'] = Configuration::get('PS_CONTACT_INFO_DISPLAY_EMAIL');
+
+        return $output.$helper->generateForm(array(
```
Comment (PierreRambaud) [inline ps_contactinfo.php]:
```suggestion
        $output[] =$helper->generateForm(array(
```

And return imploded $output.
Easier to find where the output is appended

Comment (matthieu-rolland) [reply ps_contactinfo.php]: smart ! 
(and done)

**Comment by matthieu-rolland** [comment]
I treated your suggestions @PierreRambaud , this pr is ready for re-review. 👍

**Pull request merged**